mpxplay/au_cards/au_cards.c

```diff
@@ -49,6 +58,11 @@ extern one_sndcard_info VIA82XX_sndcard_info;
 extern one_sndcard_info SBLIVE_sndcard_info;
 extern one_sndcard_info CMI8X38_sndcard_info;
 extern one_sndcard_info EMU20KX_sndcard_info;
+extern one_sndcard_info CTXFI_sndcard_info;
+extern one_sndcard_info EMU10K1X_sndcard_info;
```

Comment:
As you last mentioned Trident card driver are not included, so there might be linking errors. but We can fix later if that is the case.

Reply:
I made a mistake when merging, putting back EMU20KX_sndcard_info. EMU20KX will be replaced by CTXFI.
It is OK for now because AU_CARDS_LINK_CTXFI etc. are not defined yet.